src/SourceLocator/Type/AutoloadSourceLocator.php

```diff
@@ -95,13 +94,12 @@ class_exists($className);
      * nothing so throw an exception.
      *
      * @param string $functionName
-     * @return string
-     * @throws FunctionUndefined
+     * @return string|null
      */
     private function locateFunctionByName($functionName)
     {
         if (!function_exists($functionName)) {
-            throw new FunctionUndefined('Function ' . $functionName . ' was not already defined');
+            return null;
```

Comment:
@Ocramius this isn't relevant yet, and also won't work (we can't yet reflect internal functions - see #38) so it doesn't affect us yet, and is also irrelevant for this PR